TicTacToe/Cargo.toml

```diff
@@ -0,0 +1,5 @@
+[workspace]
```

Comment:
Do we need this file?  seems like all rust stuff should be contained in the contract folder

Reply:
These are included so CI can run `forc b` and `cargo t` in the root of the project's folder without special naming for each project's contracts. You can see this here: https://github.com/FuelLabs/sway-applications/blob/39d19079f8b4a4b147337eed33b2a9e8951023f2/.github/workflows/ci.yml#L43